### ISSUE TITLE
Catboost feature analysis plots not working

### ISSUE THREAD

**Issue by aswalin** [catboost/catboost#882]
Hi, I am trying to create plots using `get_feature_statistics` but I am getting following error
```
---------------------------------------------------------------------------
CatBoostError                             Traceback (most recent call last)
<ipython-input-61-05a1dbf7e41d> in <module>
      1 feature = 'hour_of_day'
----> 2 cb.get_feature_statistics(X_train[features_new], y_train, feature, plot=True)

/anaconda2/envs/p36workshop/lib/python3.6/site-packages/catboost/core.py in get_feature_statistics(self, data, target, feature, prediction_type, cat_feature_values, plot, max_cat_features_on_plot, thread_count)
   2220             prediction_type,
   2221             feature_type,
-> 2222             thread_count
   2223         )
   2224 

_catboost.pyx in _catboost._CatBoost._get_binarized_statistics()

_catboost.pyx in _catboost._CatBoost._get_binarized_statistics()

CatBoostError: catboost/libs/data_new/objects.cpp:502: Internal CatBoost Error (contact developers for assistance): catFeature #1, object #0: value 0xA09F575B is missing from CatFeaturesHashToString
```
For nummerical feature, I got this error:
```
CatBoostError: catboost/libs/quantized_pool_analysis/quantized_pool_analysis.cpp:143: Internal CatBoost Error (contact developers for assistance): Cannot access values of float feature #0
```
I tried it for both categorical and numerical features but both are not working. My data looks like this-

<img width="1006" alt="Screenshot 2019-06-26 at 12 17 45 PM" src="https://user-images.githubusercontent.com/30161068/60208242-9e0a1080-980c-11e9-9f25-e96ad18cee01.png">

**Comment by annaveronika** [comment]
Thanks a lot for poining this out! We've fixed this in code already, the version with the fix will be on pypi in a day or so.

**Comment by annaveronika** [comment]
fixed in 0.15.2
